STACpopulator/extensions/thredds.py

```diff
@@ -132,3 +132,18 @@ def links(self) -> list[pystac.Link]:
         url = self.access_urls[ServiceType.httpserver]
         link = magpie_resource_link(url)
         return [link]
+
```

Comment:
Same here... should this be a subclass of `ExtensionHelper`

Reply:
ExtensionHelper is a pydantic class, while the Datacube and THREDDS helpers are not. I agree it should be harmonized, but I suggest we do this in another PR.